etl/steps/data/meadow/lgbt_rights/2024-06-03/equaldex.py

```diff
@@ -0,0 +1,60 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+
+    # Retrieve snapshots.
+    snap = paths.load_snapshot("equaldex.csv")
+    snap_current = paths.load_snapshot("equaldex_current.csv")
+    snap_indices = paths.load_snapshot("equaldex_indices.csv")
+
+    # Load data from snapshots.
+    tb = snap.read()
+    tb_current = snap_current.read()
+    tb_indices = snap_indices.read()
+
+    #
+    # Process data.
+    # CURRENT DATASET
+
+    # Rename year_extraction to year
+    tb_current = tb_current.rename(columns={"year_extraction": "year"})
+
+    # Set index as country, year and issue and verify that there are no duplicates
+    tb_current = tb_current.format(["country", "year", "issue"])
+
+    # COMPLETE DATASET
+    # Drop duplicates in the index
+    # Equaldex collaborators are cleaning these duplicates, that sometimes provide different values for the same year
```

Comment:
The website and the underlying data is shown in two separate files: current and historical. Current shows the current situation around these issues, and when this status started. If I compare it with the historical dataset there are some incongruencies in this status, so that's what the Equaldex team has been fixing last year (this year the issues keep appearing, so it's problematic)